tests/testthat/test-centrality.R

```diff
@@ -147,30 +147,26 @@ test_that("authority_score survives stress test", {
   withr::local_seed(42)
 
   is.principal <- function(M, lambda) {
-    expect_equal(eigen(M)$values[1], lambda)
+    all.equal(eigen(M)$values[1], lambda)
   }
 
   is.ev <- function(M, v, lambda) {
-    expect_equal(as.vector(M %*% v), lambda * v)
+    all.equal(as.vector(M %*% v), lambda * v)
   }
 
   is.good <- function(M, v, lambda) {
```

Comment:
The original implementation is better if renamed to `expect_principal()` and `expect_ev()` . We could pass the loop index `i` as a label or something, for clearer messages.